src/app/modals/dul-modal/dul-modal.controller.js

```diff
@@ -65,11 +77,19 @@
             try{
                 JSON.parse($scope.useRestriction);
             }catch(err){
-                $scope.duplicateEntryAlert(0, "Unable to process JSON");
+                $scope.duplicateEntryAlert(0, "Unable to process Structured Limitations JSON");
```

Comment:
Can this and the previous identical try/catch block be made into a function that's called to de-dupe the code?